datahub-web-react/src/graphql/lineage.graphql

```diff
@@ -174,6 +174,258 @@ fragment lineageNodeProperties on EntityWithRelationships {
             qualifiedName
         }
         editableProperties {
+            name
+            description
+        }
+        platform {
+            ...platformFields
+        }
+        ownership {
```

Comment:
also confused why this `lineage.graphql` file is being updated? Joins aren't lineage, they are relationships, but we'll never get them in a lineage specific query

Reply:
Removed.